includes/class-request.php

```diff
@@ -208,62 +215,6 @@ protected function check_content_type() {
 		return true;
 	}
 
-	/**
-	 * Check if response is a supported content type
-	 *
-	 * @param int $code Status Code.
-	 * @param array $response Response if Present.
-	 *
-	 * @return WP_Error|true return an error or that something is supported
-	 */
-	protected function check_response_code() {
```

Comment:
I removed this check because it is very fragile. It misses a lot of valid codes, besides `200`.

* https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_success
* https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#3xx_redirection

Reply:
Besides that it should be handled by the WordPress request handler.

Reply:
I am so sorry, but I thought the error handling was still part of the `wp_safe_remote_*` methods. So yes we should re-add it 🙄